modules/core/arrow-extras-data/src/main/kotlin/arrow/data/fingertree/FingerTree.kt

```diff
@@ -111,7 +111,7 @@ sealed class FingerTree<T> {
 
   fun isEmpty() = this is Empty
 
-  fun asList(): List<T> = this.asListHelper(emptyList(), this)
+  fun asList(): List<T> = this.asSequence().toList()
```

Comment:
Why the extra jump?

Reply:
Because it is so much faster than my implementation. Unfortunately, I don't know why yet 😅

Reply:
Okay, I see what's there. Keep it!

Reply:
K. I think it's so much slower because of the list concat.